src/inmanta/protocol/rest/client.py

```diff
@@ -40,14 +40,18 @@ class RESTClient(RESTBase):
     HTTP verbs. For other methods the POST verb is used.
     """
 
-    def __init__(self, endpoint: "Endpoint", connection_timout: int = 120) -> None:
+    def __init__(self, endpoint: "Endpoint", connection_timout: int = 120, force_instance: bool = False) -> None:
         super().__init__()
         self.__end_point: "Endpoint" = endpoint
         self.daemon: bool = True
         self.token: Optional[str] = inmanta_config.Config.get(self.id, "token", None)
         self.connection_timout: int = connection_timout
         self.headers: Set[str] = set()
         self.request_timeout: int = inmanta_config.Config.get(self.id, "request_timeout", 120)
+        max_clients: int = inmanta_config.Config.get(
+            "agent_rest_transport", "max_clients", 1000
+        )  # todo what should be the default here?
```

Comment:
here I have a mypy issue. It is also present for "request_timeout" but I don't know how to fix it.
```

+ src/inmanta/protocol/rest/client.py:51: error: No overload variant of "get" of "Config" matches argument types "str", "str", "int"  [call-overload]
+ src/inmanta/protocol/rest/client.py:51: note: Possible overload variants:
+ src/inmanta/protocol/rest/client.py:51: note:     def get(cls) -> ConfigParser
+ src/inmanta/protocol/rest/client.py:51: note:     def get(cls, section: str, name: str, default_value: Optional[str] = ...) -> Optional[str]
```